tests/unit/amo/components/TestAddon.js

```diff
@@ -781,11 +781,8 @@ describe(__filename, () => {
       }),
     });
 
-    expect(root.find('.AddonDescription')).toHaveLength(1);
-
-    expect(root.find('.AddonDescription-contents')).toHaveHTML(
-      `<div class="AddonDescription-contents">${summary}</div>`,
-    );
+    expect(root.find('.AddonDescription')).toHaveLength(0);
+    expect(root.find('.AddonDescription-contents')).toHaveLength(0);
```

Comment:
Thanks for the updates @SeanPrashad : ) It's looking good. 

Just a few more notes here:

So I don't think we need both of these "expects"; If `AddonDescription` is 0 then `AddonDescription-contents` will be 0 too. I think initially we had the extra check here b/c we were checking what the text was.

We also need to update the test case's description here. I believe it's outdated now. You can probably put something similar to what the static theme test case shows (I believe it's right above this one).

Lastly, we should have similar test cases now for the other add-on types too. Like language, dictionary, search..etc.